src/test/java/teammates/test/cases/action/InstructorFeedbackQuestionEditActionTest.java

```diff
@@ -971,6 +979,102 @@ public void testExecuteAndPostProcessConstSumOption() {
 
         // All existing responses should be deleted as the options are edited
         assertTrue(frDb.getFeedbackResponsesForQuestion(fq.getId()).isEmpty());
+
+        ______TS("edit min and max points constraint -> edit failure ");
```

Comment:
Move these new test case out and form a new test method just for min/max point constraints. The old `testExecuteAndPostProcessConstSumOption()` is messy enough. 

It would be best if u can extract common code from the long long Params since we are only interested in `FEEDBACK_QUESTION_CONSTSUMPOINTS_HAS_MINPOINTS_CONSTRAINT ` `FEEDBACK_QUESTION_CONSTSUMPOINTS_HAS_MAXPOINTS_CONSTRAINT ` `FEEDBACK_QUESTION_CONSTSUMPOINTSMIN ` `FEEDBACK_QUESTION_CONSTSUMPOINTSMAX ` and the action result